public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsights.json

```diff
@@ -2491,6 +2538,6 @@
   "timepicker": {},
   "timezone": "",
   "title": "Azure App monitoring - Application Insights",
-  "uid": "Yo38mcvnz",
-  "version": 80
+  "uid": "Yo38mcvnz1",
```

Comment:
should we remove the uid?

Reply:
I personally have no issue with removing it! I know that for some dashboards we were considering linking between dashboards and in that case we would want to keep the UID, but in this case I think it's okay to just have a new one be regenerated on import.